There's compilation failure :
```
2026-04-09T23:53:50.0065280Z [ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.13.0:testCompile (default-testCompile) on project jbpm-tests: Compilation failure: Compilation failure: 
2026-04-09T23:53:50.0067693Z [ERROR] /home/runner/work/incubator-kie-kogito-runtimes/incubator-kie-kogito-runtimes/apache_incubator-kie-kogito-runtimes/jbpm/jbpm-tests/src/test/java/org/jbpm/bpmn2/IntermediateEventTest.java:[2892,40] cannot find symbol
2026-04-09T23:53:50.0069664Z [ERROR]   symbol:   class IntermediateThrowEventProjectScopeModel
2026-04-09T23:53:50.0070392Z [ERROR]   location: class org.jbpm.bpmn2.IntermediateEventTest
2026-04-09T23:53:50.0072073Z [ERROR] /home/runner/work/incubator-kie-kogito-runtimes/incubator-kie-kogito-runtimes/apache_incubator-kie-kogito-runtimes/jbpm/jbpm-tests/src/test/java/org/jbpm/bpmn2/IntermediateEventTest.java:[2892,113] cannot find symbol
2026-04-09T23:53:50.0073894Z [ERROR]   symbol:   variable IntermediateThrowEventProjectScopeProcess
2026-04-09T23:53:50.0074587Z [ERROR]   location: class org.jbpm.bpmn2.IntermediateEventTest
2026-04-09T23:53:50.0076128Z [ERROR] /home/runner/work/incubator-kie-kogito-runtimes/incubator-kie-kogito-runtimes/apache_incubator-kie-kogito-runtimes/jbpm/jbpm-tests/src/test/java/org/jbpm/bpmn2/IntermediateEventTest.java:[2898,25] cannot find symbol
2026-04-09T23:53:50.0077784Z [ERROR]   symbol:   class IntermediateThrowEventProjectScopeModel
2026-04-09T23:53:50.0078463Z [ERROR]   location: class org.jbpm.bpmn2.IntermediateEventTest
2026-04-09T23:53:50.0078991Z [ERROR] -> [Help 1]
2026-04-09T23:53:50.0080536Z org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.13.0:testCompile (default-testCompile) on project jbpm-tests: Compilation failure
```

--- a/jbpm/jbpm-tests/src/test/bpmn/org/jbpm/bpmn2/intermediate/BPMN2-IntermediateThrowEventProjectScope.bpmn2
+++ b/jbpm/jbpm-tests/src/test/bpmn/org/jbpm/bpmn2/intermediate/BPMN2-IntermediateThrowEventProjectScope.bpmn2
@@ -18,7 +18,6 @@
   ~ under the License.
   -->
 
-<?xml version="1.0" encoding="UTF-8"?>
 <bpmn2:definitions xmlns:bpmn2="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:bpsim="http://www.bpsim.org/schemas/1.0" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" xmlns:di="http://www.omg.org/spec/DD/20100524/DI" xmlns:drools="http://www.jboss.org/drools" xmlns:xsi="xsi" id="_3dQLgBaZED-2qrq4Yj5t1Q" xsi:schemaLocation="http://www.omg.org/spec/BPMN/20100524/MODEL BPMN20.xsd http://www.jboss.org/drools drools.xsd http://www.bpsim.org/schemas/1.0 bpsim.xsd http://www.omg.org/spec/DD/20100524/DC DC.xsd http://www.omg.org/spec/DD/20100524/DI DI.xsd " exporter="jBPM Process Modeler" exporterVersion="2.0" targetNamespace="http://www.omg.org/bpmn20">
   <bpmn2:itemDefinition id="_outputItem" structureRef="String"/>
   <bpmn2:itemDefinition id="__SignalEvent_dataInputXItem" structureRef="String"/>

